backend/models/time.py

```diff
@@ -122,7 +122,8 @@ def _measure_one_to_many_points(self, origin, point_list):
                         source_points=[origin],
                         destination_points=batch)[0]
                 )
-        return list(filter(None.__ne__, measurements))
+        # To avoid errors if data is missing, replace durations with ABSURDLY_LARGE_TIME_IN_SECONDS.
+        return [measurement or ABSURDLY_LARGE_TIME_IN_MINUTES * 60 for measurement in measurements]
```

Comment:
Note that this will convert 0.0 to ABSURDLY_LARGE_TIME ;)

Reply:
True, but it is in second, what are the chances?

Reply:
```
[measurement if measurement is not None else ABSURDLY_LARGE_TIME_IN_MINUTES * 60 for measurement in measurements]
```